packages/mask/src/social-network/ui.ts

```diff
@@ -226,6 +228,19 @@ export async function activateSocialNetworkUIInner(ui_deferred: SocialNetworkUI.
     }
 }
 
+export async function loadSocialNetworkUIs(): Promise<SocialNetworkUI.Definition[]> {
```

Comment:
Why you're going to load every SNS Adaptors?

Reply:
We need to have the user authorize all SNS before creating persona.